app/utils/callback/lava.py

```diff
@@ -15,11 +15,19 @@
 import models
 import os
 import yaml
+from decimal import Decimal
 
 import utils
+import utils.tests_import as tests_import
 import utils.boot
+import utils.kci_test
 import utils.db
 import utils.lava_log_parser
+import pymongo
+import models.test_suite as mtest_suite
+import models.test_case as mtest_case
+import datetime
+import bson
 
```

Comment:
Most of the above import statements are not used in the file. Are they needed for future expansions or were they used for testing?

In case we can drop them. They are:
```
from decimal import Decimal
import utils.tests_import as tests_import
import pymongo
import models.test_suite as mtest_suite
import models.test_case as mtest_case
import datetime
import bson
```